packages/terra-table/src/TableRow.module.scss

```diff
@@ -0,0 +1,77 @@
+/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
```

Comment:
Could we add this around the block that specifically need this stylelint disable?

Reply:
Updated.